R/cv_varsel.R

```diff
@@ -770,14 +782,9 @@ loo_varsel <- function(refmodel, method, nterms_max, ndraws,
         if (no_psis_eval) {
           if (getOption("projpred.warn_psis", TRUE)) {
             warning(
-              "In the recalculation of the reference model's PSIS-LOO CV ",
-              "weights for the performance evaluation, the number of draws ",
-              "after clustering or thinning is too small for Pareto ",
-              "smoothing. Using standard importance sampling (SIS) instead. ",
-              "Watch out for warnings thrown by the original-draws Pareto ",
-              "smoothing to see whether it makes sense to increase the number ",
-              "of draws (resulting from the clustering or thinning for the ",
-              "performance evaluation). Alternatively, K-fold CV can be used."
+              "Using standard importance sampling (SIS), as the number of draws",
+              "or clusters is too small for PSIS. For improved accuracy reliability",
```

Comment:
Is "accuracy reliability" supposed to read only "accuracy" (or "reliability")?

Reply:
fixed

Reply:
thanks